packages/pkl.impl.ghactions/steps/SetupPkl.pkl

```diff
@@ -34,7 +34,7 @@ fixed step: Workflow.Step = new {
   shell = "bash"
   run =
     """
-    DIR=$(mktemp /tmp/pkl-\(version)-XXXXXX)"
+    DIR="$(mktemp /tmp/pkl-\(version)-XXXXXX)"
```

Comment:
Note: I prefer to add these variables in as env vars, and reference them using shell references. Then, adding language injection (adding `// language=bash`) so that IntelliJ can highlight this and point out obvious errors to me.

Also, I like moving things into a separate script file and `read()`ing them in if they are long enough.